docs/conf.py

```diff
@@ -78,14 +78,14 @@
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 
-__import__(setup_cfg['package_name'])
-package = sys.modules[setup_cfg['package_name']]
-
-# The short X.Y version.
-version = package.__version__.split('-', 1)[0]
-# The full version, including alpha/beta/rc tags.
-release = package.__version__
+from pkg_resources import get_distribution, DistributionNotFound
+try:
+    __version__ = get_distribution(__name__)
+except DistributionNotFound:
+    __version__ = 'unknown'
```

Comment:
I'm going to work on trying to simplify the sphinx ``conf.py`` file in the template, so in future I think a lot of this will be easier.